Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -597,6 +597,23 @@ - (void) setTrackingDisabled:(BOOL)disabled {
     }
 }
 
+- (BOOL)sendCloseRequests {
+    @synchronized(self) {
+        NSNumber *b = (id) [self readObjectFromDefaults:@"sendCloseRequests"];
+        if ([b isKindOfClass:NSNumber.class]) return [b boolValue];
+        
+        // by default, we send close requests
+        return true;
+    }
+}
+
+- (void)setSendCloseRequests:(BOOL)disabled {
+    @synchronized(self) {
+        NSNumber *b = [NSNumber numberWithBool:disabled];
```

Comment:
`NSNumber *b = @(disabled);`? Or even just `[self writeObjectToDefaults: @(disabled)];`? Not that it really matters.

Reply:
updated